org_status/export/Exporter.py

```diff
@@ -0,0 +1,15 @@
+class Exporter:
+    NAME = None
+    OUTPUT_FILE = None
+
+    def save_org_repos_to_file(self, repos):
```

Comment:
Nope. We instead need a method to convert an org list from various hosts into a format specific dump string (in you case a YAML string).